.golangci.yml

```
@@ -0,0 +1,71 @@
+
+linters:
+  enable:
+    # Default linters
+    - errcheck      # Check for unchecked errors
+    - gosimple      # Simplify code
+    - govet         # Go vet checks
+    - ineffassign   # Detect unused variable assignments
+    - staticcheck   # Static analysis checks
+    - typecheck     # Type checking
+    - unused        # Find unused code
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)